targets/targets.json

```diff
@@ -4061,6 +4057,13 @@
             "network-default-interface-type": "WIFI"
         }
     },
+    "REALTEK_RTL8195AM": {
+        "supported_form_factors": ["ARDUINO"],
+        "inherits": ["MCU_RTL8195A"],
+        "detect_code": ["4600"],
+        "extra_labels": ["Realtek", "AMEBA", "RTL8195A", "RTW_EMAC"],
+        "device_has": ["ANALOGIN", "ANALOGOUT", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "SERIAL", "SPI", "TRNG", "FLASH"]
```

Comment:
Shouldn't MCU define these ? I imagine ANALOGIN supported for that MCU. If a new derived target does not implement analogin for specific reason, it could remove it. Same for extra_labels - `Realtek` is specific for this target or can be applied to MCU_ one ?

Reply:
"device_has" will move to the under MCU. So do "extra_labels". "extra_labels_add" will add to the specific target.